airbyte-webapp/src/packages/cloud/services/AppServicesProvider.tsx

```diff
@@ -43,6 +43,10 @@ const ServiceOverrides: React.FC<React.PropsWithChildren<unknown>> = React.memo(
 
   const { cloudApiUrl } = useConfig();
 
+  if (!cloudApiUrl) {
+    throw new MissingConfigError("Missing required configuration cloudApiUrl");
+  }
```

Comment:
Another option would be to log the error with our `AppMonitoringService()` instead of throwing it here. The downside is that this would not be surfaced to the user in environments without datadog.